src/LogMonitor.js

```diff
@@ -72,6 +72,10 @@ export default class LogMonitor extends Component {
 
   shouldComponentUpdate = shouldPureComponentUpdate;
 
+  state = {
```

Comment:
Oh you're totally right. Can't believe i didn't notice the reducer.js and action.js..